packages/core-mobile/app/new/routes/(signedIn)/(modals)/accountSettings/addRecoveryMethods/verifyMfa/selectMfaMethod.tsx

```diff
@@ -0,0 +1,63 @@
+import { Text, View } from '@avalabs/k2-alpine'
+import { useRouter } from 'expo-router'
+import { useSeedlessManageRecoveryMethodsContext } from 'features/accountSettings/context/SeedlessManageRecoveryMethodsProvider'
+import { RecoveryMethodList } from 'features/onboarding/components/RecoveryMethodList'
+import { RecoveryMethod } from 'features/onboarding/hooks/useAvailableRecoveryMethods'
+import { useRegisteredRecoveryMethods } from 'features/onboarding/hooks/useRegisteredRecoveryMethods'
+import React, { useState, useEffect, useCallback } from 'react'
+import { ScrollView } from 'react-native-gesture-handler'
+import SeedlessService from 'seedless/services/SeedlessService'
+import { MFA } from 'seedless/types'
+
+const TITLE = 'Verify recovery method'
+
+const SelectRecoveryMethodScreen = (): React.JSX.Element => {
+  const { navigate } = useRouter()
+  const [mfaMethods, setMfaMethods] = useState<MFA[]>([])
+  const registeredRecoveryMethods = useRegisteredRecoveryMethods(mfaMethods)
+  const { onVerifyFido } = useSeedlessManageRecoveryMethodsContext()
+  useEffect(() => {
+    const getMfaMethods = async (): Promise<void> => {
+      const mfas = await SeedlessService.session.userMfa()
+      setMfaMethods(mfas)
+    }
+    getMfaMethods()
```

Comment:
looks like we are doing this a lot. what do you think about creating a react query for this and a hook called `useMfaMethods`?